exporter/awsemfexporter/mapwithexpiry/map_with_expiry.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mapwithexpiry
+
+import (
+	"sync"
+	"time"
+)
+
+type mapEntry struct {
+	creation time.Time
+	content  interface{}
+}
+
+// MapWithExpiry act like a map which provide a method to clean up expired entries
+type MapWithExpiry struct {
+	lock    *sync.Mutex
+	ttl     time.Duration
+	entries map[string]*mapEntry
+}
+
+func NewMapWithExpiry(ttl time.Duration) *MapWithExpiry {
+	return &MapWithExpiry{lock: &sync.Mutex{}, ttl: ttl, entries: make(map[string]*mapEntry)}
+}
+
+func (m *MapWithExpiry) CleanUp(now time.Time) {
```

Comment:
This doesn't seem to be called

Reply:
If this method isn't called because it's not needed, please remove this class :)

Reply:
This `CleanUp` method is not being called for now, but we will add a new logic in another PR to use this method to cleanup the map. This is to prevent some sparse metrics staying in the map and cause memory leak.